### PR TITLE
infra: Local image udates workflow  - rhel-10 branch version

### PR DESCRIPTION
**Pull request by M4rtinK**
This is a RHEL 10 version of the local development scripts PR, which should work fine both for Fedora and RHEL 10 development.

Marking as infra due to these scripts having having no impact on anything we ship on RHEL 10.

### CODE REVIEW

**Review by KKoukiou — APPROVED** [review]
I see this was merged at two seperate commits on master as well.  But generally when introducing new code - fix up commits should be squashed prior merging.

**Review by jkonecny12 — CHANGES_REQUESTED** [review]
Please add these files to `.structure-config` file, these are not code related scripts.

**Comment by M4rtinK** [comment]
> I see this was merged at two seperate commits on master as well. But generally when introducing new code - fix up commits should be squashed prior merging.

Sure, that's how I do this when I do fixup commit myself - in this case I was kinda trying out the Github suggested change functionality. We can certainly use it the same way - add the fixup commit/s to the most relevant existing commit.

I was just thinking that in some cases a separate commit contributed by another user could work as well ? Seems like a good way to give credit for such improvements. Anyway, lets just squash these fixup commits for now. :)

**Comment by M4rtinK** [comment]
> Please add these files to `.structure-config` file, these are not code related scripts.

Added - should I do that also on the `master` branch ? :)

**Comment by jkonecny12** [comment]
> > Please add these files to `.structure-config` file, these are not code related scripts.
> 
> Added - should I do that also on the `master` branch ? :)

Yes please.

File: .structure-config [hunk 1]
```diff
@@ -14,6 +14,8 @@ scripts/jinja-render
 scripts/makebumpver
 scripts/rhel_version.py
 scripts/rhel_version.py.j2
+rebuild_boot_iso
+update_boot_iso
```
Comment (jkonecny12) [inline .structure-config:18]: Please add it here including the directory as it is line above.

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -146,6 +146,57 @@ The resulting ISO will be stored in ``./result/iso`` directory.
...
+on external infrastructure. It uses two scripts, one called ``scripts/rebuild_boot_iso`` to build a fresh bootable installation image (boot.iso)
+from Anaconda source code on the given branch and corresponding Fedora/CentOS Stream packages. The second script, called ``scripts/update_boot_iso``
+uses the Anaconda updates image mechanism together with the ``mkksiso`` command provided by the Lorax project to very quickly
+create an updated version of the boot.iso when Anaconda code is changed. The updated boot.iso can then be booted on a VM or bare metal.
+
+The ``rebuild_boot_iso`` script
+"""""""""""""""""""""""""""""""
+
+This is just a simple script that rebuilds the boot.iso from Anaconda source code on the current branch & corresponding Fedora
+(on Fedora branches) or CentoOS Stream (on RHEL branches) packages. The script makes sure to remove the old images first
```
Comment (jkonecny12) [inline CONTRIBUTING.rst:162]:
```suggestion
(on Fedora branches) or CentOS Stream (on RHEL branches) packages. The script makes sure to remove the old images first
```

**Review by jkonecny12 — COMMENTED** [review]
Looks good to me otherwise but please fix the path first.

File: CONTRIBUTING.rst (hunk 2 above)
Comment (M4rtinK) [reply CONTRIBUTING.rst]: Fixed, thanks! :)

**Review by jkonecny12 — APPROVED** [review]

**Comment by jkonecny12** [comment]
Thanks!

**Pull request merged**